testing/integ-test/src/integTest/groovy/org/gradle/integtests/ParallelStaleOutputIntegrationTest.groovy

```diff
@@ -19,9 +19,9 @@ package org.gradle.integtests
 import org.gradle.integtests.fixtures.AbstractIntegrationSpec
 import spock.lang.Issue
 
-@Issue("https://github.com/gradle/gradle/issues/17812")
+@Issue(["https://github.com/gradle/gradle/issues/17812", "https://github.com/gradle/gradle/issues/22090"])
```

Comment:
Is this issue really relevant to this test? The linked issue talks about asciidoctor and  I don't think thats really important here.

Reply:
Yes, there is quite a tangle of issues referencing other issues here, I think it helps to point to these 2 specifically to figure out what's happening.